R/utils.R

```diff
@@ -136,3 +136,16 @@ escape_xml <- function(x, replacement = c("&amp;", "&quot;", "&lt;", "&gt;", "&a
     }
     return(file)
 }
+
+.check_trust <- function(trust, format) {
+    lifecycle::deprecate_warn(
+      when = "2.0.0",
+      what = "import(trust)",
+      details = paste0("Trust will be set to FALSE by default for ", format, ".")
+    )
+    if (isFALSE(trust)) {
+        stop(format, "files may execute arbitary code. Only load", format, "files that you personally generated or can trust the origin.", call. = FALSE)
+    }
+    NULL
```

Comment:
I used this from review example provided, but curious if an explicit `NULL` return is necessary/best practice. This would be a bit new to me.

Reply:
You can choose

``` r
is.null((function() { NULL })())
#> [1] TRUE
is.null((function() {  })())
#> [1] TRUE
is.null((function() { return() })())
#> [1] TRUE
is.null((function() { invisible() })())
#> [1] TRUE
```

<sup>Created on 2024-05-12 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>